src/main.rs

```diff
@@ -98,9 +99,24 @@ enum Command {
     },
 }
 
+#[derive(StructOpt, Debug)]
+enum Docs {
+    #[structopt(name = "build", about = "Render HTML docs locally")]
+    Build,
+
+    #[structopt(name = "format", about = "Publish HTML docs to hexdocs")]
```

Comment:
Perhaps state that this doesn't work yet (and refer to the issue, if it exists)

Reply:
That's right, this pull request is work-in-progress. More will come later